advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -202,7 +202,7 @@ private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulne
         id = id,
         references = references.map {
             VulnerabilityReference(
-                url = URI.create(it.url),
+                url = URI.create(it.url.trim()),
```

Comment:
The quoted issue also shows that sometimes the scheme is missing from the URI. Have you considered to also add "https" if the string does not start with "http" (but has "://")?

Reply:
I've had this considered and ended up opting in only for the super trivial work around of trimming, deciding against since since there are only two instance of the problem with the hope of having it fixed upstream. But as you asked for it I've just added a commit on top to implement that.